src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerAccountControllerTests/CreateAccount/Given_Cookie_Data_Is_Null/WhenICreateAnAccount.cs

```diff
@@ -36,8 +35,7 @@ public void Arrange()
 
             _orchestrator = new Mock<EmployerAccountOrchestrator>();
 
-            _owinWrapper = new Mock<IAuthenticationService>();
-            new Mock<IAuthorizationService>();
+            _owinWrapper = new Mock<IAuthenticationService>();          
```

Comment:
The namespace of these tests no longer makes sense, the Given_New_Journey... directory can be removed now.

Reply:
It has.. I forgot to change the Namespace name

Reply:
Feel free to give those two scenarios more meaningful names, whilst you're in there if you feel like it :0)

Something like `Given_We_Have_Account_Data` and `Given_We_Do_Not_Have_Account_Data` would be better. It's only annoying me, because I wrote them ;0)